lisa/util/constants.py

```diff
@@ -116,6 +116,8 @@
 SECURITY_PROFILE_CVM = "cvm"
 SECURITY_PROFILE_STATELESS = "stateless"
 
+GUEST_VM_TYPE_STANDARD = "STANDARD"
```

Comment:
Can it reuse above security profile items?

Reply:
If it's used in the libvirt platform only, it doesn't need to define in this file.

Reply:
We can remove this Variables. I dont see it being used anywhere.